cocos/3d/skinned-mesh-renderer/skinned-mesh-renderer.ts

```diff
@@ -146,43 +154,31 @@ export class SkinnedMeshRenderer extends MeshRenderer {
         }
     }
 
-    /**
-     * Notifies this renderer that an animation is usable.
-     * @internal This method only friends to skeletal animation component.
-     */
-    public notifyAnimationUsable (animation: SkeletalAnimation) {
-        if (this._animation) {
-            this._animation.removeUser(this);
-            this._animation = null;
-        }
-
-        animation.addUser(this);
-        this._animation = animation;
-    }
-
-    /**
-     * Notifies this renderer that the used animation is no longer usable.
-     * @internal This method only friends to skeletal animation component.
-     */
-    public notifyAnimationUnusable () {
-        this._animation = null;
-    }
-
     protected _updateModelParams () {
         this._update(); // should bind skeleton before super create pso
         super._updateModelParams();
     }
 
-    private _animation: SkeletalAnimation | null = null;
-
     private _tryBindAnimation () {
         const { _skinningRoot: skinningRoot } = this;
         if (!skinningRoot) {
             return;
         }
+
```

Comment:
Corresponding to https://github.com/cocos-creator/engine/pull/10006/files#diff-6b7762c60bea81084e63372b54ddae7766cb7e91ecc85de1977d3a51664bcd25R170-R181 . They should take same effect.